files/plugins/check_cis_audit.py

```diff
@@ -53,6 +54,7 @@ def _get_major_version():
         "level1_workstation": "cis_level1_workstation",
         "level2_workstation": "cis_level2_workstation",
     }
+    PROFILE_OPTIONS = list(PROFILE_MAP.values())
```

Comment:
**issue:** 

There is not `PROFILE_MAP` variable.

Reply:
`PROFILE_MAP` is defined in the previous statement, on line 51.